packages/ws/README.md

```diff
@@ -0,0 +1,122 @@
+<div align="center">
+  <br />
+  <p>
+    <a href="https://discord.js.org"><img src="https://discord.js.org/static/logo.svg" width="546" alt="discord.js" /></a>
+  </p>
+  <br />
+  <p>
+    <a href="https://discord.gg/djs"><img src="https://img.shields.io/discord/222078108977594368?color=5865F2&logo=discord&logoColor=white" alt="Discord server" /></a>
+    <a href="https://www.npmjs.com/package/@discordjs/ws"><img src="https://img.shields.io/npm/v/@discordjs/ws.svg?maxAge=3600" alt="npm version" /></a>
+    <a href="https://www.npmjs.com/package/@discordjs/ws"><img src="https://img.shields.io/npm/dt/@discordjs/ws.svg?maxAge=3600" alt="npm downloads" /></a>
+    <a href="https://github.com/discordjs/discord.js/actions"><img src="https://github.com/discordjs/discord.js/actions/workflows/test.yml/badge.svg" alt="Build status" /></a>
+    <a href="https://codecov.io/gh/discordjs/discord.js" ><img src="https://codecov.io/gh/discordjs/discord.js/branch/main/graph/badge.svg?precision=2&flag=ws" alt="Code coverage" /></a>
+  </p>
+</div>
+
+## About
+
+`@discordjs/ws` is a powerful wrapper around Discord's gateway
+
+## Installation
+
+**Node.js 16.9.0 or newer is required.**
+
+```sh-session
+npm install @discordjs/ws
+yarn add @discordjs/ws
+pnpm add @discordjs/ws
+```
+
+## Example usage
+
+```ts
+import { WebSocketManager, WebSocketShardEvents } from '@discordjs/ws';
+import { REST } from '@discordjs/rest';
+
+const rest = new REST().setToken(process.env.DISCORD_TOKEN);
+// This example will spawn Discord's recommended shard count, all under the current process.
+const manager = new WebSocketManager({
+	token: process.env.DISCORD_TOKEN,
+	intents: 0, // for no intents
+	rest,
+});
+
+await manager.connect();
+```
+
+### Specify shards
+
+```ts
+// Spawn 4 shards
+const manager = new WebSocketManager({
+	token: process.env.DISCORD_TOKEN,
+	intents: 0,
+	rest,
+	shardCount: 4,
+});
+
+// The manager also supports being responsible for only a sub-set of your shards:
+
+// Your bot will run 8 shards overall
+// This manager will only take care of 0, 2, 4, and 6
+const manager = new WebSocketManager({
+	token: process.env.DISCORD_TOKEN,
+	intents: 0,
+	rest,
+	shardCount: 8,
+	shards: [0, 2, 4, 6],
+});
+
+// Alternatively, if your shards are consecutive, you can pass in a range
+const manager = new WebSocketManager({
+	token: process.env.DISCORD_TOKEN,
+	intents: 0,
+	rest,
+	shardCount: 8,
+	shards: {
+		start: 0,
+		end: 4,
+	},
+});
+```
+
+### You can also have the shards spawned in `worker_thread`s.
```

Comment:
This heading feels out of place with the full stop at the end as others don't?

How about "Specify `worker_threads`" and state "You can also have..." underneath?